src/gsi_ncdiag/proc_gsi_ncdiag.py

```diff
@@ -92,11 +92,13 @@ def run_radiances_obsdiag(radfile, outdir):
                 help="Path to directory to output observations")
 ap.add_argument("-d", "--obsdiag_dir",
                 help="Path to directory to output observations")
-ap.add_argument("-b", "--add_obsbias", default=False,
+ap.add_argument("-b", "--add_obsbias", default=False, action='store_true',
```

Comment:
if you didn't already know, @danholdaway and @CoryMartin-NOAA this changes the arguments so instead of passing:
`-b True`

you would now pass:
`-b`

without it still defaults to false presence of the switch turns it to true. This may interrupt scripting you have in place so let me know or suggest remove of the `action='store_true'` and we can rollback